airflow/models/trigger.py

```diff
@@ -97,15 +97,15 @@ def bulk_fetch(cls, ids: Iterable[int], session: Session = NEW_SESSION) -> dict[
         Fetches all the Triggers by ID and returns a dict mapping
         ID -> Trigger instance.
         """
-        query = (
-            session.query(cls)
+        query = session.scalars(
+            select(cls)
             .filter(cls.id.in_(ids))
```

Comment:
Missed changing this (also the `all` call below should be removed

Reply:
fixed